client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -88,6 +90,74 @@ def _get_artifact_msg(self, key):
     def _get_artifact(self, key):
         raise NotImplementedError
 
+    def set_schema(self, input: dict, output: dict = None) -> None:
+        """
+        Sets the input and output schemas of this Deployable Entity. Schemas are stored as
+        model artifacts.
+
+        The output schema is optional.
+
+        To validate a prediction's input and output against these schemas, use the
+        :func:`~verta.registry.validate_schema` decorator.
+
+        Parameters
+        ----------
+        input : dict
+            Input schema as an OpenAPI-compatible json dict. Easiest to create using pydantic.BaseModel.schema() [#]_.
+        output : dict
+            Output schema as an OpenAPI-compatible json dict. Easiest to create using pydantic.BaseModel.schema().
+
+
+        References
+        ----------
+        .. [#] https://docs.pydantic.dev/1.10/usage/schema/
+
+        """
+        if not isinstance(input, dict):
+            raise TypeError(
+                f"`input` must be of type dict, not {type(input)}; did you remember to call `.schema()`?"
```

Comment:
```suggestion
                f"`input` must be of type dict, not {type(input)}; did you remember to" 
                " call `.schema()`?"
```
Nit-picky, I know, but a `black` formatted for length

Reply:
your `black` is more aggressive than mine apparently!